.lua-format

```diff
@@ -0,0 +1,33 @@
+# https://github.com/Koihik/LuaFormatter/blob/master/docs/Style-Config.md
```

Comment:
This could be a candidate for generation by pk, right?

Reply:
Yes, correct!